src/game/default/g_client_stats.c

```diff
@@ -54,6 +54,11 @@ void G_ClientToIntermission(g_entity_t *ent) {
 
 	ent->client->locals.ammo_index = 0;
 	ent->client->locals.pickup_msg_time = 0;
+	
+	// take a screenshot if we're supposed to
+	if (g_force_screenshot->integer == 1){
+		G_ClientStuff(ent, "r_screenshot\n");
+	}
```

Comment:
That's a great idea too, but there should be a way to force all active players to take a screen/demo. Having worked on all those euro q2 leagues (edl/edge/adl/etc) in a tournament setting this is a must. I'm constantly being surprised by how retarded some players can be...